### PR TITLE
feat(accessPolicy): Add Account Access Policy to rxAccountInfo

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-135

Depends on: https://github.com/rackerlabs/encore-ui-api-js/pull/60

* [x] Dev LGTM
* [x] Dev LGTM
* [x] Design LGTM
* [x] QE LGTM

Screenshot:

screen shot 2015-09-03 at 4 10 51 pm

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -8,7 +8,6 @@
     <!-- Content inside bower:css is auto-generated via `grunt wiredep` -->
     <!-- bower:css -->
     <link rel="stylesheet" href="bower_components/angular-hotkeys/build/hotkeys.min.css" />
-    <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.css" />
```
Comment (bry4n) [inline demo/index.html]: Hmm. I don't think I removed this line?

Comment (CITguy) [inline demo/index.html]: yeah, that line is kinda important

File: src/rxAccountInfo/rxAccountInfo.spec.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ describe('rxAccountInfo', function () {
 
         account = {
             name: 'Mosso',
-            status: 'Active'
+            status: 'Active',
+            accessPolicy: 'Full'
```
Comment (CITguy) [inline src/rxAccountInfo/rxAccountInfo.spec.js]: Can you add additional tests for the remaining two accessPolicy values ('Suspended', 'Terminated')?

Comment (kobowley) [inline src/rxAccountInfo/rxAccountInfo.spec.js]: @ElementE  please review for approval.

**Comment by parlarjb** [comment]
LGTM

**Comment by halkon** [comment]
LGTM

**Comment by Droogans** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Comment by kobowley** [comment]
LGTM in lieu of design Product approval

**Pull request merged**